mods/cnc/chrome/settings.yaml

```diff
@@ -508,43 +515,43 @@ Container@SETTINGS_PANEL:
 							Height: 20
 							Font: Regular
 							Text: Fetch Community News
-						Checkbox@PERFGRAPH_CHECKBOX:
+						Checkbox@SENDSYSINFO_CHECKBOX:
 							X: 310
 							Y: 70
 							Width: 300
 							Height: 20
 							Font: Regular
-							Text: Show Performance Graph
+							Text: Send system information
```

Comment:
Can you put, in the same `Tiny` font that we use for the video desc label (probably needs to be split over two lines) something like: "Your OS, OpenGL version, and and .NET runtime version, and language settings are sent with an anonymous ID to help priorise future development"?